layouts/head.html

```diff
@@ -11,6 +11,7 @@
   <link href="/css/pygments.css" media="screen" rel="stylesheet" type="text/css">
   <script src="/js/jquery.js" type="text/javascript"></script>
   <script src="/js/documentation.js" type="text/javascript"></script>
+  <script src="/js/millennium_diff_table.js" type="text/javascript"></script>
```

Comment:
Since we call `displayOverviewTable` in `documentation.js` pretty sure this file needs to be before LN 13 is added.

Reply:
Fixed the order: 2645ccc7c7b4913282d7406baee42fcf437dc1f4